content/blog/monthly-may-2020.md

```diff
@@ -0,0 +1,92 @@
+
+This month our team made 2 releases!
+
+You can review the changelogs here:
+
+*   [3.0.14](https://github.com/tikv/tikv/releases/tag/v3.0.14)
+    *   New features
+        *   Improve the performance when many write conflicts and  `BatchRollback` condition occur in optimistic transactions in [#7650](https://github.com/tikv/tikv/pull/7605)
+    *   Bug fixes
+*   [4.0.0-rc.2](https://github.com/tikv/tikv/releases/tag/v4.0.0-rc.2)
+    *   Compatibility Changes
+        *   Move the encryption-related configuration to the security-related configuration in [#7810](https://github.com/tikv/tikv/pull/7810)
+    *   New features
+        *   Support encryption debugging for tikv-ctl in [#7698](https://github.com/tikv/tikv/pull/7698)
+        *   Support encrypting the lock column family in snapshots in [#7712](https://github.com/tikv/tikv/pull/7712)
+        *   Support heatmap in the Grafana dashboard for Raftstore latency summary to better diagnose the jitter issue in [#7717](https://github.com/tikv/tikv/pull/7717)
+        *   Support setting the upper limit for the size of the gRPC message in [#7824](https://github.com/tikv/tikv/pull/7824)
+        *   Add in Grafana dashboard the encryption-related monitoring metrics in [#7827](https://github.com/tikv/tikv/pull/7827)
+        *   Support Application-Layer Protocol Negotiation (ALPN) in [#7825](https://github.com/tikv/tikv/pull/7825)
+        *   Support using the task ID provided by the client as the identifier in the unified read pool in [#7814](https://github.com/tikv/tikv/pull/7814)
+        *   Improve the performance of the batch insert request in [#7718](https://github.com/tikv/tikv/pull/7718)
+    *   Bug fixes
+
+## Reading materials
+
+In [Implement Raft in Rust](https://tikv.org/blog/implement-raft-in-rust/), [@siddontang](https://github.com/siddontang) shares the design of raft-rs, a Raft implementation in Rust designed by TiKV community, and how to develop your own app using it.
+
+In [TiKV in JD Cloud & AI](https://tikv.org/blog/tikv-in-jd-cloud-ai/), Can Cui, an Infrastructure Specialist at JD Cloud & AI, introduces how TiKV empowered JD Cloud & AI to manage huge amounts of OSS metadata with a simple and horizontally scalable architecture. 
+
+[@nick_r_cameron](https://twitter.com/nick_r_cameron) wrote [Building, Running, and Benchmarking TiKV and TiDB](https://pingcap.com/blog/building-running-and-benchmarking-tikv-and-tidb/) to introduce how to build and run your own TiDB or TiKV, and how to run some benchmarks on those databases.
+
+## Notable PRs
+
+*   [@Little-Wallace](https://github.com/Little-Wallace) supported batching multiple write requests into an entry in [#6683](https://github.com/tikv/tikv/pull/6683).
+*   [@nrc](https://github.com/nrc) enabled TiKV to split a region automatically if it contains many locks in [#7676](https://github.com/tikv/tikv/pull/7676), which can improve the performance of the large transaction if it continues writing to a single region.
```

Comment:
Please re-write this two - sounds weird to say the author supports/enables...  It's the PR that does the job